packages/main/src/Link.js

```diff
@@ -158,6 +163,15 @@ class Link extends WebComponent {
 		return LinkRederer;
 	}
 
+
+	onBeforeRendering() {
+		const needsNoReferrer = this.target === "_blank"
+			&& this.href
+			&& this._isCrossOrigin();
+
+		this._state._rel = needsNoReferrer ? "noreferrer" : undefined;
```

Comment:
this._rel, instead of this._state._rel ?

Reply:
this does not make it to the state if the _rel is not in the metadata.